internal/state/memory.go

```diff
@@ -0,0 +1,65 @@
+package state
```

Comment:
In mem store is a good interim solution but we should definitely have a plan to move away from this and into some sort of persistent storage solution. The in mem store is going to require us to keep vertically scaling the instance used to run this, not to mention potentially exceeding the memory limit causing the system to crash

Reply:
Agreed, this is called out in documentation as well